src/main/java/medicaldoctor/ControllerManager.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor;
+
+import java.util.HashMap;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.layout.StackPane;
+
+
+
+public class ControllerManager extends StackPane{
+    private HashMap<String, Node> screen = new HashMap<>();
+  
+    public ControllerManager(){
+        super();
+    }
+    
+    public void addScreen(String screenName, Node node){
+        screen.put(screenName,node);
+    }
+    
+    public Node getScrene(String screenName){
+        return screen.get(screenName);
+    }
+    
+    public void load(String name, String fxmlFile){
+        try{
+           FXMLLoader loader = new FXMLLoader(getClass().getClassLoader().getResource(fxmlFile));
+           Parent parentLoader = (Parent) loader.load(); 
+           ParentInterface pc = ((ParentInterface) loader.getController());
+           pc.setScreenParent(this);
+           addScreen(name,parentLoader);
+        }
+        catch(Exception e){
+            System.out.println("FXML file could not be loaded!");
```

Comment:
throw the exceptions, since the production copy does not have a console.

Reply:
It's there for me to read and know where certain errors are.

Reply:
but you could also throw an exception, and the message will tell you what happened. and with an exception it is more clear that something went wrong. it is less important here, I am just trying to encourage exception throwing where it is needed later to be displayed to the user.